cmd/skopeo/copy.go

```diff
@@ -131,5 +151,13 @@ var copyCmd = cli.Command{
 			Value: "",
 			Usage: "`DIRECTORY` to use to store retrieved blobs (OCI layout destinations only)",
 		},
+		cli.StringFlag{
+			Name:  "format, f",
+			Usage: "`MANIFEST TYPE` (oci, v2s1, or v2s2) to use when saving image to directory using the 'dir:' transport (default is manifest type of source)",
```

Comment:
(Non-blocking: That’s now three places that hard-code the list of options; especially this one is fairly likely to be forgotten and not updated. Consider using a single global `var formatChoices map[string]string`, and building the list of options from the map keys` at runtime.)